### DISCUSSION TITLE
Force DVC to Reference 3.0 cache locations

### DISCUSSION THREAD

**Discussion by Erotemic** [treeverse/dvc#9930] [Help]
For symlink-based caches, is there a way to force DVC to upgrade all checkout files to link to the new 3.0 cache instead of the 2.0 cache? I've run a dvc cache migrate and also a dvc checkout -R . --relink but I still have some files that point to the old .dvc/cache/xx/xxxxxxxx cache locations instead of the new .dvc/cache/files/md5/xx/xxxxxxx. I'd like to just remove the old cache entirely and have a shiney new 3.0 file structure

I don't have a MWE ATM, but will attempt to create one if I get some time.

**Comment by dberenbaum** [comment]
Hi, sorry for the lack of response in Discord. We are looking into making 2.0 files get migrated when doing `dvc commit -f`, but it doesn't work yet. You can keep track in #9818.
